content/en/cloud_cost_management/recommendations/_index.md

```diff
@@ -0,0 +1,134 @@
+
+{{< img src="cloud_cost/recommendations/resource_side_panel.png" alt="Cloud cost allocation table showing requests and idle costs over the past month on the Containers page" style="width:100%;" >}}
+
+Recommendations are run on a daily basis, and are automatically refreshed in your account. When new recommendations are released, Datadog automatically adds them to your account. 
+
+### Terminate resource recommendations
+
+Datadog scans your cloud environment to identify and surface orphaned resources that can be removed. 
+
+Low Utilization EC2 Instances
+: EC2 instances with less than 5% CPU utilization, and less than 10% memory utilization. 
+
+Unattached EBS Volumes Unattached
+: Volumes that have been detached from an EC2 instance. 
+
+Unused EBS Volumes
+: Volumes attached to a non-running EC2 instance.
+
+Unused RDS Instances
+: RDS instances with 0 database connections and 0 replica lag.
+
+Incomplete S3 Multipart Uploads
+: Incomplete multipart uploads (requires [Storage Lens metrics][6]).
+
+Unused Redshift Cluster
+: Redshift cluster with 0 database connections.  
+
+Unused Elasticache Redis Cluster
+: Elasticache Redis Cluster has 0 cache hits and 0 replication bytes.  
+
+Unused MQ Broker
+: An MQ broker with 0 connections. 
+
+Old ECR Images
+: ECR Image bytes are older than 180 days. 
+
+OpenSearch Cluster
+: An OpenSearch cluster with 0 connections.
+
+Unused Classic Elastic Load Balancers
+: Classic Elastic Load Balancer with no active connections that is not attached to an EC2 instance.
+
+Unused Network Elastic Load Balancer
+: A network load balancer with 0 processed bytes. 
+
+Unused NAT Gateway
+: A NAT Gateway that has no bytes sent out. 
+
+Unused Elastic IP Address
+: Elastic IP addresses with idle charges in your AWS cost and usage report.
+
+Application Load Balancer (ELB)
+: An application load balancer with no traffic being processed for the past 7 days.
+
+### Migrate resource recommendations
+
+Datadog surfaces resources that are running on legacy hardware, which you can consider upgrading to reduce your costs and improve the performance of your resources.
+
+Legacy EC2 Instance
+: EC2 Instances that are previous generation, and can be upgraded to a newer instance type. 
+
+GP2 EBS Volumes
+: EBS volumes that are GP2 and can be upgraded to GP3 for cost reduction and performance improvement.  
+
+I01 EBS Volumes
+: EBS volumes that are I01 and can be upgraded to GP3 for cost reduction and performance improvement.
+
+### Rightsize resource recommendations
+
+Datadog identifies individual resources that are under-utilized or over-provisioned, which you can consider adjusting the size and configuration to reduce your costs and improve the performance of your resources.
+
+Overprovisioned EC2 instances
```

Comment:
Maybe use "overprovisioned" throughout the page instead of "over-provisioned" for consistency? Webster says "underutilized," so you would be able to update that one too if you wanted.

Reply:
We do call this "Downsize EC2 Instance" in the UI. It's a bit inconsistent with the others which are in fact "over-provisioned"

Reply:
```suggestion
Downsize EC2 instances
```